src/main/java/life/qbic/App.java

```diff
@@ -37,12 +38,23 @@ public static void main(String[] args) throws IOException {
    */
   private static QbicDataDownloader loginToOpenBIS(
       PostmanCommandLineOptions commandLineParameters) {
-    LOG.info(String.format("Please provide password for user '%s':", commandLineParameters.user));
 
-    String password = OpenBISPasswordParser.readPasswordFromConsole();
+    String password;
+    if (!commandLineParameters.passwordEnvVariable.isEmpty()) {
+
+      password = OpenBISPasswordParser.readPasswordFromEnvVariable(commandLineParameters.passwordEnvVariable, commandLineParameters.user);
```

Comment:
As discussed, it would be better to not make getting the password depending on the user information.